### PR TITLE
Fizzbuzz with if, for done

### LINKED ISSUE #1: Do fizzbuzz with if, for
## Task list

- [x] do fizz
- [x] do buzz
- [x] do fizzbuzz

### PR DESCRIPTION
**Pull request by ulgoon**
close #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,16+1):
+    if i%15==0:
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:2]: Could you do this with 1 if?

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,16+1):
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:1]: set num range 16 to 31

**Review by kingwangzzang1234 — CHANGES_REQUESTED** [review]

File: fizzbuzz.py (hunk 1 above)
Comment (ulgoon) [reply fizzbuzz.py:2]: No ploblem^^

File: fizzbuzz.py (hunk 2 above)
Comment (ulgoon) [reply fizzbuzz.py:1]: Doooooone!

**Review by kingwangzzang1234 — APPROVED** [review]
Well done!

**Pull request merged**